apps/client/src/hooks/api/qna/useGetQnaList.ts

```diff
@@ -0,0 +1,25 @@
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { get } from '@/libs/api/client';
+
+export type GetQnaListResponse = {
+  id: number;
+  title: string;
+  content: string;
+  category: string;
+  createTime: string;
+  likeCount: number;
+  commentCount: number;
+};
+
+export const QNA_LIST_QUERY_KEY = 'qnaList';
+
+export const useGetQnaList = (categories?: string[]) => {
+  const qnaListEndPoint = categories ? `api/qna?category=${categories}` : 'api/qna';
+
+  const qnaListQuery = useSuspenseQuery<GetQnaListResponse[]>({
+    queryKey: [QNA_LIST_QUERY_KEY, categories],
+    queryFn: async () => await get<GetQnaListResponse[]>(qnaListEndPoint),
+  });
+
```

Comment:
```suggestion
  const qnaListQuery = useSuspenseQuery<GetQnaListResponse[]>({
    queryKey: [QNA_LIST_QUERY_KEY, categories],
    queryFn: async () => await get(qnaListEndPoint),
  });

```
위쪽에서 타입을 지정해주어서 타입이 이미 GetQnaListResponse로 읽히고 있어요 해당 타입은 불필요하다고 생각되어 삭제해요! 해당 부분은 https://github.com/sickgyun/sickgyun-client/pull/253 PR에서 수정되었어요

Reply:
넵! 수정하겠습니다.